pkg/loop/internal/types/types.go

```diff
@@ -44,14 +44,6 @@ type OCR3CapabilityProvider interface {
 // Relayer is like types.Relayer, but with a dynamic NewPluginProvider method.
 type Relayer interface {
 	types.ChainService
-
-	// NewChainWriter returns a new ChainWriter.
-	// The format of config depends on the implementation.
-	NewChainWriter(ctx context.Context, chainWriterConfig []byte) (types.ChainWriter, error)
-
-	// NewContractReader returns a new ContractReader.
-	// The format of contractReaderConfig depends on the implementation.
-	NewContractReader(ctx context.Context, contractReaderConfig []byte) (types.ContractReader, error)
 	NewConfigProvider(context.Context, types.RelayArgs) (types.ConfigProvider, error)
```

Comment:
Should this contain the chainService? Have to check how Relayer interface is used